mhep/mhep/assessments/views.py

```diff
@@ -196,6 +200,10 @@ def put(self, request, pk, tag):
 
 
 class ListCreateOrganisationAssessments(generics.ListCreateAPIView):
+    permission_classes = [
+        IsMemberOfOrganisation,
+        IsAuthenticated,
```

Comment:
Hmm, shouldn't these be the other way around? Fail first if they're not authenticated?